### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by omarbabou**
In the last part of the project, I achieved many things such as:

    - A good understanding of the use of figma.
    - Acquire important knowledge on the use of form validation in HTML.
    - Implement better form validation for a good user experience.
    - Fix linters error.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,202 +42,281 @@ <h1>
...
+            placeholder="Write your message here"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required
+          ></textarea>
+          <div class="btn">
+            <button type="submit">Get In Touch</button>
+          </div>
+        </form>
```
Comment (keroloussamy) [inline index.html]: - [x] Kindly when I tried to submit, the form doesn't get submitted. 
You should use [formspree service](https://formspree.io/html) to submit. 
something like the 
- `action` value should be similar to `https://formspree.io/f/{form_id}`
- `method` value equals `POST`

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @omarbabou, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (omarbabou) [reply index.html]: Thank you for the feedback.

**Review by gemmen29 — APPROVED** [review]
Hi @omarbabou, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**